src/blocks/shared/CommonMessage.scss

```diff
@@ -0,0 +1,104 @@
+.common-message {
+  --_common-message-padding: var(--common-message-padding, var(--bloom-s4));
```

Comment:
why do these all start with `_`?

Reply:
That's essentially because we have a mix here of "private" variables and public variables. If `--common-message--padding` is set, the `--_common-message-padding` private variable will mirror that, otherwise it defaults to the second argument there which is the `--bloom-s4` variable. We haven't needed this pattern before, but we use it here so that the alert/toast/etc. components can override the common set of tokens if necessary.

Reply:
Why couldn't we just do `--common-message-padding: var(--bloom-s4);` ?

Reply:
@emilyjablonski If you look at the stylesheets for alert, message, and toast, they're all setting `--common-message-*` values, except that the component-specific tokens they're referencing actually don't exist (unless overridden by a consumer). These "missing" tokens then get assigned to `--_common_message-*` tokens, which will roll over to all of the default values (`--bloom-s4`, etc.). This enables consumers to override tokens for just alert, or just toast, etc., all while we only have to set the default values once for all three. Consumers could also just override common tokens if they want to change all three simultaneously.

If that's too much abstraction or whatever, I'm definitely open to feedback.

Reply:
I think I might prefer this flow if you think it still covers all situations: CommonMessage has variables like `--common-message-padding: var(--bloom-s4);` and Alert has variables like --alert-padding: var(--common-message-padding, var(--bloom-s4));` - wdty?

Reply:
@emilyjablonski Unfortunately that particular way of doing it ends up with circular logic due to how CSS variables resolve in real time. The common message tokens can't be set to alert tokens which are set to common message tokens. But I did just check in an update with unwinds some of the DRY stuff so it's each component just has its own set of tokens. That's probably fine, unless we think downstream consumers will want to make lots of changes to alert, message, and toast all all once.